### ISSUE TITLE
New Command - Test AG Health

### ISSUE THREAD

**Issue by sql-aus-hh** [dataplat/dbatools#8508]
### Summarize Command's Functionality

Maybe we need a command to check the Availability Group Health of a SQL Server, an Instance, or a specific AG?
For troubleshooting purposes or any monitoring activities?

### Is there a command that is similiar or close to what you are looking for?

No

**Comment by potatoqualitee** [comment]
yeah! we've wanted this for a while. it was mentioned here https://github.com/dataplat/dbatools/issues/4556 and i think @tboggiano had wanted one too

**Comment by andreasjordan** [comment]
Hi Björn! Can you give a little more detail? Like what would be the minimum feature that has to be implemented in a first version. If you have some code to test the health of you AGs, then share it here and I can implement the command.

**Comment by andreasjordan** [comment]
Closing this as we are currently working on https://github.com/dataplat/dbatools/issues/9911
